### DISCUSSION TITLE
Synthetic data

### DISCUSSION THREAD

**Discussion by bhack** [keras-team/keras-cv#360] [General]
https://venturebeat.com/2021/12/21/taking-the-world-by-simulation-the-rise-of-synthetic-data-in-ai/

Do we have a more general strategy about  syntheric data in CV other then augmentation?

See also:
https://github.com/google-research/kubric

**Comment by bhack** [comment]
One use case, as we have already now with KLPs, is to interact in the training loop with a policy that control simulations/image rendering  parameters. I suppose that if our position is that we could produce these offline with the engine I suppose that a similar approach could be valid also for the classical KLPs (e.g. to  produce the augmented dataset offline with OpenCv/Pillow/etc.. instead of writing our own KLPs layers..)
Also it could required when we want to have differentiable approaches https://github.com/keras-team/keras-cv/issues/41#issuecomment-1024588287

Some other interesting opensource (enterprise level) engines are landing: 
https://github.com/o3de/sig-simulation/blob/main/governance/sig-simulation-charter.md

/cc @qlzh727 @LukeWood

**Comment by LukeWood** [comment]
Yeah, synthetic data is a super interesting (and useful!) space.  I've also been seeing projects begin to land.  

From my perspective, our first priority is to tackle classification and bounding box detection with state of the art results on the traditional datasets (imagenet, COCO, openimages, etc). I don't think we're super far off.  After that I think it makes sense to begin expanding into more cutting edge spaces such as synthetic data generation, domain adaptation, etc.

**Comment by innat** [comment]
## SHIFT: A Synthetic Driving Dataset for Continuous Multi-Task Domain Adaptation

This is cool. 

![image](https://user-images.githubusercontent.com/17668390/175574563-4fb468ac-f029-41ef-8d8c-a577d5a68778.png)

Paper: [SHIFT](https://openaccess.thecvf.com/content/CVPR2022/papers/Sun_SHIFT_A_Synthetic_Driving_Dataset_for_Continuous_Multi-Task_Domain_Adaptation_CVPR_2022_paper.pdf)
Project page: https://www.vis.xyz/shift/

https://user-images.githubusercontent.com/17668390/175573991-97897748-04dc-42c2-af81-5db14e54a44b.mp4

**Reply by innat** [reply]
- Weather conditions (including cloudiness, rain, and fog intensity)
- Time of day

![image](https://user-images.githubusercontent.com/17668390/175575280-60193f7a-4fb9-4b00-98d2-72ac88eb2d2f.png)

![image](https://user-images.githubusercontent.com/17668390/175575407-e799b17f-62ed-4e83-881b-56b2601cc3a0.png)
